dcos/dcos-ui
# DCOS-21270: Create service puts cursor at wrong position in app id box

Put cursor at the end of inputbox when creating a service.

Closes https://jira.mesosphere.com/browse/DCOS-21270

Solution taken from https://coderwall.com/p/0iz_zq/how-to-put-focus-at-the-end-of-an-input-with-react-js .

## Testing
1. Go to services
2. Create a single-container service
3. Verify that the cursor is at the end of the input after "/".
4. Repeat for pod.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
![peek 2018-12-07 10-33](https://user-images.githubusercontent.com/40791275/49636776-3f19b100-fa0c-11e8-9fcf-7dd6c9a4461e.gif)
